strings-csharp/ReadStringFromResourceFile/ReadStringFromResourceFile/Classes/ResourcesManager.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Globalization;
+using System.Reflection;
+using System.Resources;
+
+namespace ReadStringFromResourceFile;
+
+public class ResourcesManager
+{
+    private Assembly? assembly;
```

Comment:
This field is only assigned to and never used, it can be removed

Reply:
Should had been:
 this.assembly = assembly ?? Assembly.GetExecutingAssembly();
on the following line, I forget that.
I´ve renamed all private fields with prefix underscore to get rid of this issue.
Also, in the Program.cs, I changed a little bit, now it shows an error for the LOGOFF_2 resource, which doesn´t exist. I´m going to update the article also.